clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -220,7 +223,9 @@ public class ConsumerConfig extends AbstractConfig {
             "partition of the fetch is larger than this limit, the " +
             "batch will still be returned to ensure that the consumer can make progress. The maximum record batch size " +
             "accepted by the broker is defined via <code>message.max.bytes</code> (broker config) or " +
-            "<code>max.message.bytes</code> (topic config). See " + FETCH_MAX_BYTES_CONFIG + " for limiting the consumer request size.";
+            "<code>max.message.bytes</code> (topic config). See " + FETCH_MAX_BYTES_CONFIG + " for limiting the consumer request size. " +
+            "Consider increasing this limit especially in the cases of remote storage reads (KIP-405), because currently only " +
```

Comment:
> Consider increasing this limit

Could this be changed to?
```
Consider increasing the <code>max.partition.fetch.bytes</code> limit ...
```

If the user increases the `fetch.max.bytes` from 50 MB, then it might cause GC pressure due to holding of large continuous byte array.

Reply:
good point